fvm/src/kernel/default.rs

```diff
@@ -200,18 +205,24 @@ where
     }
 
     fn set_root(&mut self, new: Cid) -> Result<()> {
+        crate::assert_validator!(self, "Validator can't set root.");
```

Comment:
Shouldn't this be a local macro?

Reply:
macro is used in syscalls::charge_gas as well, though it could be made local and have that other one just be manually written.